lib/rollbar/notifier.rb

```diff
@@ -203,7 +203,9 @@ def critical(*args)
 
     def enabled?
       # Require access_token so we don't try to send events when unconfigured.
-      configuration.enabled && configuration.access_token && !configuration.access_token.empty?
+      configuration.enabled &&
+      configuration.access_token &&
```

Comment:
I don't know what's common in Ruby, but I've always seen this type of multi-line statements with some indentation after the first line.

Reply:
While I think this formatting is more common in Ruby, there are definitely projects that indent multi-line expressions. As noted above, we could choose to follow the other convention and update the rubocop config to match.